social/social.go

```diff
@@ -500,6 +528,148 @@ func (c *Client) GetSteamProfile(ctx context.Context, publisherKey string, appID
 	return profileWrapper.Response.Params, nil
 }
 
+func (c *Client) CheckAppleToken(ctx context.Context, bundleId string, idToken string) (*AppleProfile, error) {
```

Comment:
We should dump the `idToken` into DEBUG level logs to help diagnose any issues with the input sent by game clients.

Reply:
Doable, but we should talk this through first.